### PR TITLE
remove cluster.local

### PR DESCRIPTION
**Pull request by moosecanswim**
**Please provide enough information so that others can review your pull request:**

1. Explain the **details** for making this change. What existing problem does the pull request solve? If it resolves an existing issue, be sure to use a Github [keyword](https://help.github.com/en/articles/closing-issues-using-keywords) to automatically close it.

This PR removes `.cluster.local` from each host definition that calls a service (`x.namespace.svc.cluster.local`) .  This change allows the service mesh to be deployed into a cluster with a custom dns (service ex: `x.namespace.svc.testdecipher` would be the service inside a cluster called testdecipher).

To test:
- launch ec2 or local minikube
- install helm
- install gm2
- ensure the dashboard is visable
- (optional) deploy service in another namespace and check to see if you can make a connection to a service in the default namespace without adding `cluster.local`. 
- (optional) The service i deployed was shell demo (https://github.com/DecipherNow/openshift-development/blob/master/shell.yaml).  Once it is deployed exec into it and install telnet.  run the command `telnet redis.default.svc 6379`

you should see:
```
Trying 172.17.0.12...
Connected to redis.default.svc.cluster.local.
```
from here you can see that `.cluster.local` was automatically appended. 

2. What **changes to custom.yaml** are required?

none

3. Have you documented any additional setup steps or configurations options?

none

4. Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.

https://18.216.181.118:31690/#/grey-matter-data-v1-00?ascending=true&configPaneOpen=false&configPaneService=&displayType=Cards&groupByAttribute=Status&searchQuery=&sortByAttribute=Name

image

### CODE REVIEW

**Comment by chrisbsmith** [comment]
@tilleryd since this is a relatively major change to our charts, I think it prudent to create tag after this PR is approved.  Thoughts?

**Comment by tilleryd** [comment]
@chrisbsmith agreed, a minor version increment is warranted

**Comment by moosecanswim** [comment]
@tilleryd @chrisbsmith, i rebased this pr due to fix the merge issue and also incremented the charts patch version for each chart.  I also updated the local dev requirements file for parity with the one pointing to our hosted charts.

File: greymatter/requirements-dev.yaml [hunk 1]
```diff
@@ -1,36 +1,47 @@
 dependencies:
   - name: catalog
-    repository: 'file://../catalog'
-    version: '2.0.1'
+    repository: "file://../catalog"
+    version: "2.0.6"
```
Comment (chrisbsmith) [inline greymatter/requirements-dev.yaml]: @moosecanswim we should keep these versions in sync with `requirements.yaml`

Comment (moosecanswim) [reply greymatter/requirements-dev.yaml]: i had this on my local but missed it on the push

**Comment by moosecanswim** [comment]
@chrisbsmith I squashed the commits.  could you re-approve?

**Pull request merged**